commit 443468df8df85139ccf6ec9dec34ad0dbcf4f861
Merge: 9593f09 831016b
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Wed Apr 1 02:13:30 2015 -0700

    Merge pull request #1771 from AnHardt/iss1770
    
    Added new - changed entries from Fix up LCD_PROGRESS_BAR

diff --git a/Marlin/ultralcd.h b/Marlin/ultralcd.h
index c2529d1..a89c262 100644
--- a/Marlin/ultralcd.h
+++ b/Marlin/ultralcd.h
@@ -99,7 +99,8 @@
 #else //no LCD
   FORCE_INLINE void lcd_update() {}
   FORCE_INLINE void lcd_init() {}
-  FORCE_INLINE void lcd_setstatus(const char* message) {}
+  FORCE_INLINE void lcd_setstatus(const char* message, const bool persist=false) {}
+  FORCE_INLINE void lcd_setstatuspgm(const char* message, const uint8_t level=0) {}
   FORCE_INLINE void lcd_buttons_update() {}
   FORCE_INLINE void lcd_reset_alert_level() {}
   FORCE_INLINE void lcd_buzz(long duration,uint16_t freq) {}
